### ISSUE TITLE
Summary for included files of a firmware ocntains the firmware itself.

### ISSUE THREAD

**Issue by maringuu** [fkie-cad/FACT_core#1182]
Simple reproduction should be to upload a Linux kernel as firmware.
Then the software components plugin will recognize the whole firmware to be a kernel (which is correct).

Now clicking on "load summary for included files" shows the same result again, despite it being in the file itself rather than an included file.

**Comment by jstucke** [comment]
I agree that this doesn't make sense. It could break some existing scripts, though. Better we discuss this in the next "quarterly"
